modules/otel-config-instrumentation.adoc

```diff
@@ -98,9 +98,45 @@ spec:
 
 |===
 
+[id="otel-configuration-of-opentelemetry-sdk-variables_{context}"]
+== Configuration of the OpenTelemetry SDK variables
+
+You can use the `instrumentation.opentelemetry.io/inject-sdk` annotation in the OpenTelemetry Collector custom resource to instruct the {OTELName} Operator to inject the following OpenTelemetry SDK environment variables into your pod:
+
+* `OTEL_SERVICE_NAME`
+* `OTEL_TRACES_SAMPLER`
+* `OTEL_TRACES_SAMPLER_ARG`
+* `OTEL_PROPAGATORS`
+* `OTEL_RESOURCE_ATTRIBUTES`
+* `OTEL_EXPORTER_OTLP_ENDPOINT`
+* `OTEL_EXPORTER_OTLP_CERTIFICATE`
+* `OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE`
+* `OTEL_EXPORTER_OTLP_CLIENT_KEY`
+
+.Values for the `instrumentation.opentelemetry.io/inject-sdk` annotation
+[cols="a, a"]
+|===
+| Value | Description
+
+|`"true"`
```

Comment:
The instrumentation CR name is also considered in the annotation. 

**"true" - to inject Instrumentation resource with default name from the current namespace.**
"my-instrumentation" - to inject Instrumentation CR instance with name "my-instrumentation" in the current namespace.
"my-other-namespace/my-instrumentation" - to inject Instrumentation CR instance with name "my-instrumentation" from another namespace "my-other-namespace".
"false" - do not inject

Reply:
Thank you, Ishwar. It's on the next several lines.